SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -283,8 +304,7 @@ private Task HandleUserConfirmationAsync(UserConfirmation userConfirmation)
                 foreach (var key in keys)
                 {
                     _unprocessedManagedErrorLogs.Remove(key);
-                    ErrorLogHelper.RemoveStoredErrorLogFile(key);
-                    // TODO: Remove exception files
+                    RemoveAllStoredErrorLogFiles(key);
```

Comment:
Not quite related to your changes, but do we really need to loop? Can just clear all error files

Reply:
What if we crash during processing in another thread? (I checked, no lock). I guess it's safer that way to allow saving the new files.